packages/Php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/array_dim_fetch.php.inc

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+function arrayDimFetch()
+{
+    reset([1][0]);
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+function arrayDimFetch()
+{
+    reset([1][0]);
+}
+
+?>
```

Comment:
If nothing changes, you can remove this whole part. 

Also, the fixture file should start with `skip_*`/`keep_*` to make it clear for future debugging.